electrum/plugins/keepkey/qt.py

```diff
@@ -310,6 +321,23 @@ def pin_matrix_widget_class(self):
         from keepkeylib.qt.pinmatrix import PinMatrixWidget
         return PinMatrixWidget
 
+    @hook
+    def init_wallet_wizard(self, wizard: 'QENewWalletWizard'):
+        self.extend_wizard(wizard)
+
+    # insert keepkey pages in new wallet wizard
+    def extend_wizard(self, wizard: 'QENewWalletWizard'):
+        super().extend_wizard(wizard)
+        views = {
+            'keepkey_start': {'gui': WCScriptAndDerivation},
+            'keepkey_xpub': {'gui': WCHWXPub},
+            'safet_not_initialized': {'gui': WCKeepkeyInitMethod},
+            'safet_choose_new_recover': {'gui': WCKeepkeyInitParams},
+            'safet_do_init': {'gui': WCKeepkeyInit},
```

Comment:
`safet`->`keepkey`? copy-paste

Reply:
weird.. I'm pretty sure I fixed this, as well as ecdsa's patch below. Must be git stashed instead of committed.